.circleci/config.yml

```diff
@@ -277,6 +281,8 @@ jobs:
       - run:
           name: Build tarball
           command: |
+            source "${BASH_ENV}"
+            cat "${BASH_ENV}"
```

Comment:
do we need to cat the envs?

Reply:
this was for debugging - you're right, don't need to.